### DISCUSSION TITLE
Create SQS queue without DLQ

### DISCUSSION THREAD

**Discussion by otezz** [getlift/lift#387] [Constructs]
Is there a way to create SQS queue without dlq? Or can I create fifo queue with non fifo dlq?

**Comment by mnapoli** [comment]
Yes you can, don't create a DLQ.

**Reply by otezz** [reply]
Hi @mnapoli thank you for your response.

Could you please elaborate more on how I can create a Queue without DLQ? I've checked the [documentation](https://github.com/getlift/lift/blob/master/docs/queue.md), it says the `queue` construct will deploy a DLQ by default, but no mention on the documentation on how one can disable creation of DQL

**Reply by mnapoli** [reply]
oh in that case you don't use Lift at all and create it using CloudFormation in serverless.yml
